src/app/doi.org/page.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import apolloClient from 'src/utils/server/apolloClient'
+
+import { Row, Col } from 'src/components/Layout'
+import Error from 'src/components/Error/Server'
+import ExampleText from 'src/components/ExampleText/ExampleText'
+import SearchWork from 'src/components/SearchWork/Server'
+import { SEARCH_DOI_QUERY, QueryData, QueryVar } from 'src/data/queries/searchDoiQuery'
+
+interface Props {
+  searchParams: SearchParams
+}
+
+interface SearchParams extends Partial<QueryVar> {
+  filterQuery?: string
+}
+
+export default async function SearchDoiPage ({ searchParams }: Props) {
+  const { query, filterQuery, ...vars } = searchParams
+
+  // Show examply text if there is no query
+  if (!query || query === '') return (
+    <ExampleText>
+      <div>
+        Search works by keyword(s) or DOI.<br /><br />
+
+        Examples:
+        <ul>
+          <li><a href="/doi.org?query=climate+change">climate change</a></li>
```

Comment:
should we change these to <Links> ? Since they are internal to the application?

Reply:
Yes we should